packages/js/product-editor/src/components/header/hooks/use-save-draft/use-save-draft.tsx

```diff
@@ -86,6 +87,18 @@ export function useSaveDraft( {
 				}
 			);
 
+			const statusMap: {
```

Comment:
I think this can be moved outside of the `useSaveDraft` function since its value does not change.